crowd_anki/config/config_settings.py

```diff
@@ -0,0 +1,74 @@
+from enum import Enum
+from dataclasses import dataclass
+
+from aqt import mw
+from ..utils.constants import USER_FILES_PATH
+
+@dataclass
+class ConfigSettings():
+    class Properties(Enum):
+        SNAPSHOT_PATH = "snapshot_path"
+        AUTOMATED_SNAPSHOT = "automated_snapshot"
+        SNAPSHOT_ROOT_DECKS = "snapshot_root_decks"
+        EXPORT_DECK_SORT_METHODS = "export_deck_sort_method"
+        EXPORT_DECK_SORT_REVERSED = "export_deck_sort_reversed"
```

Comment:
probably should have the "note" in the name instead of "deck". Something like `EXPORT_NOTE_SORT_METHODS` and `EXPORT_NOTES_REVERSE_ORDER`

Reply:
I begrudgingly agree :smirk: Will change this soon